.NET/Microsoft.Recognizers.Text.Number/Extractors/BasePercentageExtractor.cs

```diff
@@ -200,14 +200,14 @@ protected static ImmutableHashSet<Regex> BuildRegexes(HashSet<string> regexStrs,
         }
 
         /// <summary>
-        /// get the number extractor results and convert the extracted numbers to @sys.num, so that the regexes can work
+        /// get the number extractor results and convert the extracted numbers to @sys.num, so that the regexes can work.
         /// </summary>
-        /// <param name="str"></param>
-        /// <param name="positionMap"></param>
-        /// <param name="numExtResults"></param>
-        /// <returns></returns>
-        private string PreprocessStrWithNumberExtracted(string str, out Dictionary<int, int> positionMap,
-            out IList<ExtractResult> numExtResults)
+        /// <param name="str">sentence to proccess.</param>
+        /// <param name="positionMap">position Map.</param>
+        /// <param name="numExtResults">number extractor result.</param>
+        /// <returns>return according type "builtin.num" or "builtin.num.percentage".</returns>
+        private string PreprocessStrWithNumberExtracted(
+            string str, out Dictionary<int, int> positionMap, out IList<ExtractResult> numExtResults)

```

Comment:
Either put them all in the same line like 
` private void PostProcessing(List<ExtractResult> results, string originSource, Dictionary<int, int> positionMap, IList<ExtractResult> numExtResults)`
or one per line
```
 private void PostProcessing(
            List<ExtractResult> results, 
            string originSource, 
            Dictionary<int, int> positionMap,
            IList<ExtractResult> numExtResults)
```